src/Common/Storage/Commands.Storage/Common/Cmdlet/NewAzureStorageAccountSasToken.cs

```diff
@@ -0,0 +1,143 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.WindowsAzure.Commands.Storage.Common.Cmdlet
+{
+    using System;
+    using System.Management.Automation;
+    using System.Security.Permissions;
+    using Microsoft.WindowsAzure.Commands.Storage.Model.Contract;
+    using Microsoft.WindowsAzure.Storage;
+
+    [Cmdlet(VerbsCommon.New, StorageNouns.AccountSas), OutputType(typeof(String))]
+    public class NewAzureStorageAccountSasTokenCommand : StorageCloudBlobCmdletBase
+    {
+        [Parameter(Mandatory = true, HelpMessage = "Service type that this SAS token applies to.")]
+        public SharedAccessAccountServices Service { get; set; }
+
+        [Parameter(Mandatory = true, HelpMessage = "Resource type that this SAS token applies to.")]
+        public SharedAccessAccountResourceTypes ResourceType { get; set; }
+
+        [Parameter(Mandatory = true, HelpMessage = "Permissions.")]
+        public string Permission { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Protocol can be used in the request with this SAS token.")]
+        public SharedAccessProtocol Protocol { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "IP, or IP range ACL (access control list) that the request would be accepted from by Azure Storage.")]
+        public string IPAddressOrRange { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Start Time")]
+        public DateTime? StartTime { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Expiry Time")]
+        public DateTime? ExpiryTime { get; set; }
```

Comment:
"ExpiryTime" is standard name in azure rest API, and consistent with other new SAS token cmdlets which already exist for a long time. So we don't suggest to change that.